File: uri-judge/[1866]Conta.cpp
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

#include <bits/stdc++.h>
#define desync ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL)
#define endl '\n'
using namespace std;

int main(){

	desync;
	int Casos, N;
	cin >> Casos;

	for(int i = 0; i < Casos; i++){
		cin >> N;
		if(N % 2 == 0 ){
			cout << 0 << endl;
		}
		else{
			cout << 1 << endl;
		}
	}

}

File: uri-judge/java-resolutions/[1866]Conta.java
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

import java.io.IOException;
import java.util.Scanner

public class Main {
 
    public static void main(String[] args) throws IOException {
 
       Scanner hr = new Scanner(System.in);
       int Casos, N;
       
       Casos = hr.nextInt();
       
       for(int i = 0; i < Casos; i++){
           N = hr.nextInt();
           if(N % 2 == 0){
               System.out.println("0");
           }
           else{
               System.out.println("1");
           }
       }
 
    }
 
}
